webapp/.eslintrc.json

```diff
@@ -632,5 +632,47 @@
         "onlyEquality": false
       }
     ]
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.tsx", "**/*.ts"],
+      "extends": [
```

Comment:
Here we can replace it directly with the typescript plugin setting.
```suggestion
      "extends": "plugin:@typescript-eslint/recommended",
```

Reply:
done